src/js/relay.firefox.com/get_profile_data.js

```diff
@@ -152,6 +152,9 @@
           );
         });
 
+        console.log("localAddressCache/matchingLocalAddress?.used_on", matchingLocalAddress?.used_on);
```

Comment:
question: Do we normally leave logging calls?

Reply:
Nope! Removed in 83da7dff3e3b1bd59e1c9af1d86de25111521534

Reply:
No. Removed in 83da7dff3e3b1bd59e1c9af1d86de25111521534